bibimainapp/pom.xml

```diff
@@ -20,48 +21,59 @@
             <url>https://jitpack.io</url>
         </repository>
     </repositories>
-    
-    
+
+
     <dependencies>
-        
+
         <!-- internal dependecies -->
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>bibimaininterfaces</artifactId>
-            <version>1.1</version>        
+            <version>1.1</version>
         </dependency>
-        
+
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>bibiservsearch</artifactId>
-            <version>1.1</version>        
+            <version>1.1</version>
+        </dependency>
+
+        <dependency>
```

Comment:
Why we need an extra dependency to an older Spring Security version (3.0.1 instead if 3.0.8) ?